spec.bs

```diff
@@ -1243,7 +1243,7 @@ For now, see [https://github.com/wicg/observable#operators](https://github.com/w
              ignore>value</var>.
 
           : [=internal observer/error steps=]
-          :: 
+          ::
```

Comment:
My IDE auto trimmed this whitespace, arguably unrelated to this change — so happy to revert if so.